salt/modules/inspectlib/collector.py

```diff
@@ -257,7 +257,10 @@ def _save_payload(self, files, directories, links):
         for p_type, p_list in (
             ("f", files),
             ("d", directories),
-            ("l", links,),
+            (
+                "l",
+                links,
+            ),
```

Comment:
This is dumb

Reply:
Look at the lines above this to see what I mean

Reply:
Fixed